shuup/front/views/category.py

```diff
@@ -63,7 +63,7 @@ def get_product_filters(self):
         return {
             "shop_products__shop": self.request.shop,
             "variation_parent__isnull": True,
-            "shop_products__categories": self.object,
+            "shop_products__categories__id__in": self.object.get_descendants().values_list("pk", flat=True),
```

Comment:
Does this include the current category or only children?

Reply:
You were correct, it included only children. Patched it to include current category too.